ccp/lib/utils/authorization.py

```diff
@@ -0,0 +1,24 @@
+"""
+This file contains authorization handlers
+"""
+
+
+class Authorization(object):
+
+    def __init__(self):
+        self.authorization_header_string = ""
+
+    def add_header(self, headers):
+        """
+        Adds appropriate authorization headers to request headers
+        :param headers: The headers dict
+        :type headers dict
+        """
+        headers["Authorization"] = self.authorization_header_string
+
+
+class BearerAuthorization(Authorization):
+
+    def __init__(self, token):
+        super(BearerAuthorization, self).__init__()
```

Comment:
I would initialize the parent class constructor with value `"Bearer %s" % token` instead of overriding the variable of parent class.